src/lib/components/ui/menu/menu-item.svelte

```diff
@@ -12,5 +12,6 @@
 	variant="ghost"
 	dimension={dimension ?? store.size}
 	disabled={disabled ?? store.disabled}
+	tabindex={disabled || store.disabled || !store.open ? -1 : 0}
```

Comment:
The `tabindex` should come from props.

```suggestion
	tabindex={disabled || store.disabled || !store.open ? -1 : tabindex}
```